### PR TITLE
Babel in the generator

### PR DESCRIPTION
**Pull request by apechimp**

### CODE REVIEW

**Comment by apechimp** [comment]
CR @lanetix/engineers

**Comment by todesignandconquer** [comment]
:wink:

File: app/index.js [hunk 1]
```diff
@@ -155,15 +155,19 @@ module.exports = yeoman.generators.Base.extend({
...
         'express@4',
+        'heroku-node@1',
         '@lanetix/microservice@2'
       ], { save: true })
 
       this.npmInstall([
-        'mocha',
+        'babel-eslint@4',
         'istanbul',
+        'isparta',
```
Comment (todesignandconquer) [inline app/index.js]: THIS
IS
PARTA

**Comment by armw4** [comment]
Looks g to me 🌚

**Pull request merged**